bbb/runner.py

```diff
@@ -23,8 +23,17 @@ def main():
     logfile = config[args.service[0]]["logfile"]
     if not logfile:
         logfile = "%s.log" % args.service
+    debug_logfile = "debug-%s" % logfile
```

Comment:
Are you sure you want this logged to a separate file? It may make it harder to read if you need to look at debug+non-debug messages.